apps/src/applab/applab.js

```diff
@@ -763,6 +769,10 @@ Applab.onMouseMoveDebugResizeBar = function (event) {
                        Math.min(MAX_DEBUG_AREA_HEIGHT,
                                 (window.innerHeight - event.pageY) - offset));
 
+  if (debugAreaController.isShut()) {
+    debugAreaController.snapOpen();
```

Comment:
There's a minimum height on the open debug area.  If you drag from closed, it snaps open to the minimum size, and will start increasing in size once your cursor is above the resize bar.

Josh L. suggested a threshold-based pullout like iOS sometimes has, where it drags smoothly up from closed but if it's below the minimum height when you release it will fall back down.  That sounds awesome, but maybe not worth the implementation time at the moment.  I figured it could be a separate PR if we decide to do it.